### PR TITLE
Added Storage Capacity Tracking feature to Powerstore docs

### PR DESCRIPTION
**Pull request by AkshaySainiDell**
# Description
Added Storage Capacity Tracking feature to Powerstore docs

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/483 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/features/powerstore.md [hunk 1]
```diff
@@ -717,7 +717,7 @@ spec:
 
 >Note: Default description value is `pvcName-pvcNamespace`. 
 
-The following is the list of all the attribtues supported by PowerStore CSI driver: 
+The following is the list of all the attributes supported by PowerStore CSI driver:
```
Comment (rsedlock1958) [inline content/docs/csidriver/features/powerstore.md:720]: Change "The following" to "This"

File: content/docs/csidriver/features/powerstore.md [hunk 2]
```diff
@@ -730,3 +730,17 @@ The following is the list of all the attribtues supported by PowerStore CSI driv
 >Make sure that the attributes specified are supported by the version of PowerStore array used. 
 
 >Configurable Volume Attributes feature is supported with Helm.
+
+## Storage Capacity Tracking
+CSI PowerStore driver version 2.5.0 and above supports Storage Capacity Tracking.
+
+This feature helps the scheduler to make more informed choices about where to start pods which depend on unbound volumes with late binding (aka "wait for first consumer"). Pods will be scheduled on a node (satisfying the topology constraints) only if the requested capacity is available on the storage array.
+If such a node is not available, the pods stay in Pending state, i.e, not scheduled.
```
Comment (rsedlock1958) [inline content/docs/csidriver/features/powerstore.md:738]: Replace comma at after "Pending state" with a period ("."). Replace "i.e, not scheduled." with "This means they are not scheduled."

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. There are a couple of minor edits required.

**Review by spriya-m — APPROVED** [review]

File: content/docs/csidriver/features/powerstore.md (hunk 1 above)
Comment (AkshaySainiDell) [reply content/docs/csidriver/features/powerstore.md]: Done.

File: content/docs/csidriver/features/powerstore.md (hunk 2 above)
Comment (AkshaySainiDell) [reply content/docs/csidriver/features/powerstore.md]: Done.

**Review by spriya-m — APPROVED** [review]

**Review by adarsh-dell — APPROVED** [review]

**Comment by adarsh-dell** [comment]
@AkshaySainiDell Do we need to merge this changes to release-1.5 branch instead of main?

**Review by spriya-m — CHANGES_REQUESTED** [review]
Good point @adarsh-dell . The changes should be merged to release-1.5 branch and not to main. 
@AkshaySainiDell - Please close this PR and raise a new PR for release-1.5 branch.

**Comment by shanmydell** [comment]
@AkshaySainiDell : I have changed the base

**Pull request closed without merging**